src/model.cpp

```diff
@@ -83,10 +83,10 @@ void srdf::Model::loadVirtualJoints(const urdf::ModelInterface& urdf_model, XMLE
     vj.type_ = std::string(type);
     boost::trim(vj.type_);
     std::transform(vj.type_.begin(), vj.type_.end(), vj.type_.begin(), ::tolower);
-    if (vj.type_ != "planar" && vj.type_ != "floating" && vj.type_ != "fixed")
+    if (vj.type_ != "planar" && vj.type_ != "floating" && vj.type_ != "fixed" && vj.type_ != "diff_drive")
     {
-      CONSOLE_BRIDGE_logError("Unknown type of joint: '%s'. Assuming 'fixed' instead. Other known types are 'planar' "
-                              "and 'floating'.",
+      CONSOLE_BRIDGE_logError("Unknown type of joint: '%s'. Assuming 'fixed' instead. Other known types are 'planar', "
+                              "'diff_drive' and 'floating'.",
```

Comment:
I would prefer to write this [as a find](https://github.com/ros-planning/moveit/commit/ae7d1ad1c5eb2bdd65511fe9a4e8574b0f77241d#diff-e92ccfc53250ee197d6201c9fca926d16c628aa027b2e5b43bb24a0602724a03R84-R90) too, but feel free to leave it as-is.